<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmlBoxModel::SettingsDialog</class>
 <widget class="QWidget" name="QmlBoxModel::SettingsDialog">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_window">
     <property name="title">
      <string>User Interface</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_style">
        <property name="text">
         <string>Style:</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_style</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBox_style"/>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_propertyEditor">
          <property name="icon">
           <iconset theme="preferences-system">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_themes">
        <property name="text">
         <string>Apply theme:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_themes"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_hideOnFocusOut">
        <property name="text">
         <string>Hide on focus out:</string>
        </property>
        <property name="buddy">
         <cstring>checkBox_hideOnFocusOut</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBox_hideOnFocusOut">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_center">
        <property name="text">
         <string>Always center Albert:</string>
        </property>
        <property name="buddy">
         <cstring>checkBox_center</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBox_center">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_onTop">
        <property name="text">
         <string>Always on top:</string>
        </property>
        <property name="buddy">
         <cstring>checkBox_onTop</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkBox_onTop">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_hideOnClose">
        <property name="text">
         <string>Hide on close:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="checkBox_hideOnClose"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_clearOnHide">
        <property name="text">
         <string>Clear on hide:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="checkBox_clearOnHide"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
